fcl/dunefd/gen/supernova/prodmarley_nue_spectrum_dune10kt_1x2x2.fcl

```diff
@@ -0,0 +1,68 @@
+#include "services_dune.fcl"
```

Comment:
Can this fhicl be simplified by inheriting from another fhicl, for example, the 1x2x6 fhicl?

Reply:
Agreed, this looks like it could be significantly reduced by inheriting from an upstream job fcl

Reply:
I have simplified it by inheriting from 1x2x6 as @lpaulucc suggested.